Sources/UIScrollView+indicatorsInternal.swift

```diff
@@ -34,44 +59,43 @@ internal extension UIScrollView {
         )
     }
 
-    internal var shouldLayoutHorizontalScrollIndicator: Bool {
+    private var shouldLayoutHorizontalScrollIndicator: Bool {
         return showsHorizontalScrollIndicator && contentSize.width > bounds.width
     }
 
-    internal var shouldLayoutVerticalScrollIndicator: Bool {
+    private var shouldLayoutVerticalScrollIndicator: Bool {
         return showsVerticalScrollIndicator && contentSize.height > bounds.height
     }
 
-
-    internal func layoutScrollIndicatorsIfNeeded() {
+    func layoutScrollIndicatorsIfNeeded() {
```

Comment:
We have a convention of always writing `internal` to make it clear that that’s our intention. It’s the default in Swift but since we’re making a public-facing library our default is public/open.

Reply:
Yeah, I was wondering why the `internals` are explicitly there. Guess I was on the wrong side of https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence :d 

brought them back now